### ISSUE TITLE
Unable to allow extension to Accessibility without elevating privileges

### ISSUE THREAD

**Issue by arekdist** [github/CopilotForXcode#236]
Trying to add "GitHub Copilot for Xcode" to enterprise management in Jamf and I cannot construct the right PPPC profile so to enable "GitHub Copilot for Xcode Extension" valid system permissions.
To be honest I even have no idea what should be enabled? Allowing Accessibility does not apply at all and documentation in https://github.com/github/CopilotForXcode/blob/main/TROUBLESHOOTING.md#extension-permission doesn't help much.
How such a profile for this app should look like, can someone enlighten me?
